docs/stake-pool-course/handbook/grafana-dashboard-tutorial.md

```diff
@@ -0,0 +1,492 @@
+---
+id: grafana-dashboard-tutorial
+title: Grafana Dashboard Tutorial
+sidebar_label: Grafana Dashboard Tutorial
+description: "Stake pool course: Grafana Dashboard Tutorial"
+image: ./img/og-developer-portal.png
+---
+
+Once you have sucessfully set-up a Cardano pool, then comes the most beautifull part - setting up your Dashboard and Alerts! You surely dont want to gaze the PC whole day, or?
+
+
+
+I found it difficult to get all the information at one place so am writing this documentaion and hopeully it helps others as well. This tutorial is for education and learning purpose only.
```

Comment:
typo (documentation, hopefully)

Also, perhaps the document should not use first person view ( "I/We found ..." ) , can take out the sentence of needed

Reply:
ok